src/service/video-manager-impl.js

```diff
@@ -106,6 +106,33 @@ export class VideoManager {
     /** @private {!../service/viewport/viewport-impl.Viewport} */
     this.viewport_ = Services.viewportForDoc(this.ampdoc);
 
+    /**
+     * Authors must include the `amp-video-docking` script for instantiation of
+     * the docking service, but it's installed in case the script is not
+     * present.
+     * This supports the rare case of very outdated documents created when the
+     * extension was not required.
+     * @private @const {function()}
+     */
+    this.maybeInstallDockingExtension_ = once(() => {
```

Comment:
I don't think it is worth having this given the new validation rules would invalidate those docs anyway and feature was experimental.